README.md

```diff
@@ -176,6 +144,97 @@ GTO the artifact file is committed to Git.
 
 </details>
 
+### Unassigning a stage
+
+Sometimes you need to mark an artifact version no longer ready for a specific
+consumer, and maybe signal a downstream system about this. You can use
+`gto untag` for that:
+
+```console
+$ gto untag awesome-model --version v0.0.1 --stage prod
+Created git tag 'awesome-model#prod#2!'
+```
+
+<details summary="Some details and options">
+
+GTO creates a special Git tag in a standard format:
+`{artifact_name}#{stage}#{e}!`.
+
+Note, that later you can create this stage again, if you need to, by calling
+`$ gto tag` again.
+
+You also may want to delete the git tag instead of creating a new one. This is
+useful if you don't want to keep extra tags in you Git repo, don't need history
+and don't want to trigger a CI/CD or another downstream system. For that, you
+can use:
+
+```console
+$ gto untag awesome-model --version v0.0.1 --stage prod --delete
```

Comment:
this could be:
`gto untag awesome-model --stage prod --from-version v0.0.1 --delete`